.pytest_cache/README.md

```diff
@@ -0,0 +1,8 @@
+# pytest cache directory #
+
+This directory contains data from the pytest's cache plugin,
+which provides the `--lf` and `--ff` options, as well as the `cache` fixture.
+
+**Do not** commit this to version control.
```

Comment:
😉  please do not commit any of the `.pytest_cache` directory.

Reply:
Sorry about that! Didn't notice at first!